tests/RobotFramework/tests/cumulocity/firmware/firmware_operation.robot

```diff
@@ -1,34 +1,28 @@
 *** Settings ***
 Resource            ../../../resources/common.resource
-Library             Cumulocity
 Library             ThinEdgeIO
+Library             Cumulocity
 
-Test Setup          Custom Setup
+Suite Setup         Custom Setup
```

Comment:
Why do we need to change to a `Suite Setup`, it would be better to leave it untouched as `Test Suite` is less prone to unexpected side effects from preceding tests.

```suggestion
Test Setup         Custom Setup
```